sdk-api-src/content/winperf/nc-winperf-pm_collect_proc.md

```diff
@@ -59,23 +59,20 @@ The **CollectPerformanceData** function is a placeholder for the application-def
 
 Type: **LPWSTR**
 
-Null-terminated string that contains the query string (for example, "Global" or "238") passed to the [RegQueryValueEx](/windows/desktop/api/winreg/nf-winreg-regqueryvalueexa) function. For the query string format, see [Using the Registry Functions to Consume Counter Data](/windows/desktop/PerfCtrs/using-the-registry-functions-to-consume-counter-data).
+Null-terminated string that contains the query string (for example, "Global" or "238") passed to the [RegQueryValueEx](/windows/desktop/api/winreg/nf-winreg-regqueryvalueexa) function. For a list of possible values for *lpValueName*, see [Using the Registry Functions to Consume Counter Data](/windows/desktop/PerfCtrs/using-the-registry-functions-to-consume-counter-data).
```

Comment:
I'm not implying that one is better that the other, but there are places in the documentation where I copied this phrase from. For better consistency we should either leave it as it is, or change elsewhere as well, I think.

Reply:
We can address updates to the other topics in a separate PR.